### DISCUSSION TITLE
How to understand "IS EMPTY"

### DISCUSSION THREAD

**Discussion by xl-better** [vesoft-inc/nebula#5954] [Q&A]
I found that there is not much info about "IS EMPTY" in nebula docs, i think it's used for variables which does not exist, and "IS NULL" is used for variables which exists but with invalid values.I am not sure if this understanding is correct, can someone help answer this?

**Comment by czpmango** [comment]
`NULL` is a **value** on the property site, and `EMPTY` reveals that there is **no value** on the property site.
